### DISCUSSION TITLE
Released Notes Document

### DISCUSSION THREAD

**Discussion by MayurUgale123** [eclipse-birt/birt#1196] [Q&A]
HI Team,
I am looking for the Released Notes Document of the BIRT Version 4.9
Specifically I am looking for the Java version supported in this release. (Please provide the Document along with answer)

**Comment by speckyspooky** [comment]
With BIRT 4.9 Java 11 is required or higher. The version will not run with Java 8.

**Reply by speckyspooky** [reply]
Please see #1185
